runner/run_benchmark.py

```diff
@@ -175,10 +180,14 @@ def _main(configuration: str, agent_name: str, max_prompt_size: Optional[int], y
         logging.warning("Running without a maximum prompt size.")
     else:
         print(f"Maximum prompt size: {max_prompt_size}")
-    agent = get_chat_session(agent_name, max_prompt_size=max_prompt_size)
+
+    agent = get_chat_session(agent_name, max_prompt_size=max_prompt_size, dataset_name=config['run_name'])
```

Comment:
Alright, I see that you're using the right value, just the variables have the wrong name. It should be `run_name` instead of `dataset_name` all the way down to the agent interface.

Reply:
I can refactor that!